spec/services/accept_unconditional_offer_spec.rb

```diff
@@ -33,6 +33,30 @@
     expect(state_change_double).to have_received(:accept_unconditional_offer!)
   end
 
+  describe 'other choices in the application' do
+    it 'with offers are declined' do
+      application_choice = create(:application_choice, :with_offer)
+      application_form = application_choice.application_form
+      other_choice_with_offer = create(:application_choice, :with_offer, application_form: application_form)
+
+      described_class.new(application_choice: application_choice).save!
+
+      expect(other_choice_with_offer.reload.status).to eq('declined')
+    end
+
+    it 'that are pending provider decisions are withdrawn' do
```

Comment:
```suggestion
    it 'withdraws applications pending provider decisions' do
```

Reply:
@cwrw amended here https://github.com/DFE-Digital/apply-for-teacher-training/pull/5107/commits/2bf1c5bb9596ad855f818ef295c339a8caf64f5a